src/style.css

```diff
@@ -0,0 +1,173 @@
+/* stylelint-disable no-descending-specificity */
+@import url("https://fonts.googleapis.com/css2?family=Roboto");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html,
+body {
+  font-family: 'Roboto', sans-serif;
+  height: 100%;
+  font-size: 16px;
+}
+
+.to-do-list {
+  list-style-type: none;
+  width: 400px;
+}
+
+#top-list {
+  z-index: 1;
+}
+
+#to-do-list {
+  overflow-y: scroll;
+}
+
+#to-do-list li {
+  padding: 0 0 0 10px;
+}
+
+.checkbox {
+  border: 2px solid gray;
+  border-radius: 2px;
+  padding: 5px;
+}
+
+.checkmark {
+  width: 14px;
+  display: none;
+}
+
+.remove {
+  width: 14px;
+  display: none;
+  margin-right: 5px;
+}
+
+.checkbox,
+.checkmark,
+#enter,
+#refresh,
+.remove {
+  cursor: pointer;
+}
+
+.task-description {
+  margin: 0 12px;
+  font-weight: 100;
+  word-break: break-all;
+  padding: 10px 0;
+}
+
+.list-item {
+  display: flex;
+  padding: 10px 0 10px 10px;
+  align-items: center;
+  justify-content: center;
+}
+
+.todo-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  justify-items: center;
```

Comment:
"Justify items" only gets considered in grid layout. It gets ignored when using flexbox

Check this out on [justify items mdn article](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-items) for more details.

Reply:
I removed it thanks for catching that.